support-frontend/assets/pages/contributions-landing/components/ContributionFormFields.tsx

```diff
@@ -115,18 +121,20 @@ function ContributionFormFields(props: PropTypes) {
 					type="email"
 					autoComplete="email"
 					supporting="example@domain.com"
-					onInput={props.updateEmail}
+					onChange={(e) => updateEmail(e.target.value)}
 					pattern={emailRegexPattern}
 					error={
 						checkoutFormHasBeenSubmitted && !checkEmail(email)
 							? 'Please provide a valid email address'
-							: null
+							: undefined
 					}
 					disabled={isSignedIn}
 				/>
 			</div>
-			<Signout isSignedIn />
```

Comment:
There's a couple of things going on here that made me a little confused.

First by attempting to just pass the name of a boolean variable, this should be the same as 

```ts
<Signout isSignedIn={true} />
```

which isnt' what we want. But, the component is actually getting its `isSignedIn` prop from `mapStateToProps`, which overrides what's passed from the parent anyway. So I've deleted the prop here, but nothing changes in the behaviour